05_iigo/index.tex

```diff
@@ -477,6 +477,40 @@ \subsection{End of Session Actions}
     \item The Judge has the \emph{power} to decide to hold an election for a new President.
 \end{enumerate}
 
+\section{Executive branch implementation}
+
+The executive branch has been implemented to facilitate the use of powers given to the president. Because any agent could potentially become the president, implementation of the executive branch has been split into two interconnected parts:
+\begin{itemize}
+\item Client-side implementation of presidential functions, which map directly to the powers given to the president.
+\item Server-side implementation of executive branch actions, which calls the presidential functions only on the currently residing president.
+\end{itemize}
+
+\subsection{Client-side implementation}
+\label{sub:president:client-side}
+Each agent can become a president therefore each agent has to implement all of the presidential functions according to the common interface. Those functions directly correspond to the powers of the executive branch outlined in Section~\ref{sec:executive} as well as accountability and transfer-of-power obligations outlined in Section~\ref{sec:accountability}. 
+
+The presidential functions can be divided into two categories:
+\begin{enumerate}
+\item Presidential power functions:
+    \begin{itemize}
+        \item \texttt{SetTaxationAmount} allows the president to decide the amount of individual taxation set to each island.
```

Comment:
Should we mention the return types of the functions? I can help describe the flow of information between the server and the client perhaps

Reply:
hm, good point!

Reply:
Also I meant "It can help describe the..." :D sorry about that